lib/test_utils.go

```diff
@@ -0,0 +1,1479 @@
+// +build e2e_test
```

Comment:
Maybe we can call this file `lib/library_test_utils.go`?

Reply:
Sounds IMHO redundant.

Reply:
Oops, already changed it, can we agree on one or the other? 

let's vote on this comment for it!
⚡️ -> lib/library_test_utils.go
❤️ -> lib/test_utils.go